app/src/main/java/com/matheus/androidudemycourse/feature/my_annotations/MyAnnotationsFragment.kt

```diff
@@ -0,0 +1,56 @@
+package com.matheus.androidudemycourse.feature.my_annotations
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.app.AppCompatActivity
+import com.matheus.androidudemycourse.R
+import com.matheus.androidudemycourse.databinding.FragmentMyAnnotationsBinding
+import com.matheus.androidudemycourse.utils.VisibilityActionEnum
+import com.matheus.androidudemycourse.utils.changeStatusBarColor
+import com.matheus.androidudemycourse.utils.handleActionBarVisibility
+
+
+class MyAnnotationsFragment : Fragment() {
+
+    private lateinit var binding: FragmentMyAnnotationsBinding
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        binding = FragmentMyAnnotationsBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        setupView()
+
+        binding.backHomeImageView.setOnClickListener {
+            activity?.onBackPressed()
+        }
+
+
+    }
+
+    private fun setupView() {
+        changeStatusBarColor(R.color.dark_brown)
+
+        handleActionBarVisibility(VisibilityActionEnum.HIDE)
+        with(binding) {
```

Comment:
gostei desse trem ai depois me ensina

Reply:
Claro

Reply:
esse with(binding) fica mais legal quando tem varios itens, ai vc n precisa ficar digitando binding varias vezes, mas nesse caso q so tem a toolbar, acho que ficaria mais simples usar apenas o binding direto, economizaria algumas linhas de codigo

Reply:
Entendi, foi força do hábito kkk